docs/tutorials/particle-filtering.md

```diff
@@ -1,3 +1,564 @@
 # Primer on Particle Filtering with Beluga
 
-Coming soon!
+:::{figure} ../_images/beluga_tutorial.gif
+:alt: Short video of a beluga_tutorial's record.
+:::
+
+## Overview
+
+Modern robotic systems operate in unstructured, unpredictable environments, making robust software crucial to handle diverse situations.
+A key challenge is uncertainty, which occurs when robots lack critical task information. Probabilistic robotics addresses this by
+representing uncertainty explicitly through probability theory, rather than relying on a single best guess.
+
+The robot localization problem is one of the most important ones in mobile robotics, as it is a fundamental requirement for autonomous navigation. The Monte Carlo Localization (MCL) algorithm is one of the most widely used algorithms for map-based localization and has become a key component in many robotic systems [^BelugaPaper]. **TODO: It is necessary to put here the reference, or just include it at the end?**
```

Comment:
I don't think this part of the overview is necessary. This should be covered by other sections in the documentation.